orttraining/orttraining/python/ort_trainer.py

```diff
@@ -727,11 +677,13 @@ def _init_session(self):
 
     def _init_onnx_model(self, inputs):
         if self.onnx_model_ is not None:
```

Comment:
`_init_onnx_model` is called on `train_step` and `eval_step` only, so at this point, either 1) `self.onnx_model_` was already provided on `ORTTrainer` constructor and post processed by `self._init_session` (line 631) and it doesn't need to be postprocessed again (line 680-681 seems redundant) or 2) `self.torch_model_` was specified on `ORTTrainer` and will be processed by the `convert_model_loss_fn_to_onnx` call below (line 686)